tests/specs/login.setup.ts

```diff
@@ -0,0 +1,18 @@
+import { test as setup } from '@playwright/test';
+import { loginPage } from '../pages/loginPage';
+import { staticData } from '../helper/staticData';
+
+const authFile = './tests/specs/playwright/.auth/user.json';
```

Comment:
So let me see if I get it right. You suggest getting rid of the setup.ts (which creates the user, logs in and store the storage info) and instead, login before each test? That can be done easily in a before hook, and if it's preferred, it can be done easily. The setup.ts is meant to make the running of the tests much faster so the time for the execution can be a lot longer. But if there are no doubts, then let's do it!